### PR TITLE
Rubens Proposal

### PR DESCRIPTION
**Pull request by RubenTerzian**
This is my humble proposal. I expect you will accept it)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -86,6 +86,18 @@ <h3 class="ba-skill__title">dreams.</h3>
         </div>
 
     </main>
+    <footer>
+        <div class="footer__wraper">
+            <form action="">
+                <input type="text" placeholder="Emaile">
```
Comment (AnnaPudlo) [inline index.html]: This is not important, but I think, there is a mistake in placeholder text. Rewrite, please.

File: index.html [hunk 2]
```diff
@@ -86,6 +86,18 @@ <h3 class="ba-skill__title">dreams.</h3>
...
+                <button>Submit</button>
+            </form>
+            <ul>
+                <li><a href="mailto:example@gmail.com">example@gmail.com</a></li>
+                <li><a href="tel:+380974737123">+380974737123</a></li>
```
Comment (AnnaPudlo) [inline index.html:97]: I hope that this phone number is not real.

**Review by AnnaPudlo — COMMENTED** [review]
It would be better, if I could see style changes on deployment preview. Can you add gh-pages?

File: index.html (hunk 2 above)
Comment (RubenTerzian) [reply index.html:97]: Sure,  this is a non-existent number

File: index.html (hunk 1 above)
Comment (RubenTerzian) [reply index.html]: Done. Please check. Is everything all right now?

**Pull request merged**